.github/workflows/validate_plan_environments.yml

```diff
@@ -0,0 +1,46 @@
+name: TF Validate and Plan for all environments
+
+# runs workflow on push and if any changes are detected in any .tf files
+on:
+  workflow_call:
+    inputs:
+      commit-identifier:
+        description: "SHA or tag to validate and plan terraform, (optional) when omitted, the head of the selected branch is used"
+        type: string
+        required: true 
+      terraform-workspace:
+        description: Terraform workspace that is used
+        type: string
+        required: true
+      terraform-root:  
```

Comment:
I know I asked for this in the validate_plan_tf file. I think we have to leave it off here.

Reply:
So I should get rid of determining terraoform_root in validate_plan_tf?